docs/deploy.md

```diff
@@ -95,6 +95,17 @@ Follow the installation guide at the [Readme page](https://github.com/Microsoft/
 
 6. For more options over the deployment, follow the [Development guide](https://github.com/Microsoft/presidio/blob/master/docs/development.md)
 
+### Expose Presidio publicly
+We recommend exposing Presidio using a DNS name.
```

Comment:
We recommend exposing it? since when?
We actually recommend deploying it within a VPN in your subscription.
The reason we deploy it publically is because we wanted a simple way to connect the demo to the cluster